### PR TITLE
snk/miconkit.cpp: Add orange overlay for Space Micon Kit

### PR DESCRIPTION
**Pull request by einstein95**
Verified with these images and from Dillweed via Discord
image
image

### CODE REVIEW

**Review by cuavas — COMMENTED** [review]
If it’s just a single flat colour across the whole screen, you can tint the screen directly and avoid the need to create an extra quad.

File: src/mame/layout/smiconk.lay [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0"?>
+<!--
+license:CC0-1.0
+-->
+<mamelayout version="2">
+	<element name="overlay">
+		<rect>
+			<bounds x="0" y="0" width="1" height="240" />
+			<color red="1" green="1" blue="0.2" />
```
Comment (cuavas) [inline src/mame/layout/smiconk.lay:9]: If you’re creating a flat quad, you don’t need to use bounds – taking the default unit square keeps things simpler.

File: src/mame/layout/smiconk.lay [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+		</rect>
+	</element>
+
+	<view name="Color Overlay">
+		<screen index="0">
+			<bounds left="0" top="0" right="3" bottom="4" />
+		</screen>
+		<element ref="overlay" blend="multiply">
+			<bounds left="0" top="0" right="3" bottom="4" />
+		</element>
```
Comment (cuavas) [inline src/mame/layout/smiconk.lay:19]: You should really define things in terms of `~scr0physicalxaspect~` and `~scr0physicalyaspect~`.

File: src/mame/layout/smiconk.lay (hunk 1 above)
Comment (happppp) [reply src/mame/layout/smiconk.lay]: Right, done

File: src/mame/layout/smiconk.lay (hunk 2 above)
Comment (happppp) [reply src/mame/layout/smiconk.lay:16]: I don't know, you'd just make it harder on yourself when you want to expand the artwork

**Pull request merged**